adapter/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what's that about?

Reply:
This enables mockito to mock the protocol enum (enums are considered final by java, cf. [baeldung](https://www.baeldung.com/mockito-final)). Mocking the protocol enum was necessary to test the executeJob-method in the adapter because, since the last refactoring, the importer object is created in that enum

Reply:
kk thanks